src/test.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Pie } from 'react-chartjs-2';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
+
+interface TestProps {
+  labels: string[];
+  data: number[];
+  backgroundColors: string[];
```

Comment:
instead of getting these lists, get a `Record<string,[number,string]>`, and when you input the data and labels and such you will put `{Italy: [20, "#b91d47"], France: [20, "#00aba9"] ...}`